generator/generator_tests/transit_test.cpp

```diff
@@ -157,7 +166,7 @@ UNIT_TEST(DeserializerFromJson_Edges)
     Edge(442018445 /* stop 1 id */, 442018446 /* stop 2 id */, 345.6 /* weight */,
          72551680 /* line id */,  false /* transfer */, {} /* shape ids */)};
 
-  TestDeserializerFromJson(jsonBuffer, "edges", expected);
+  TestDeserializerFromJson(jsonBuffer, OsmIdToFeatureIdsMap(), "edges", expected);
```

Comment:
Is it possible to implement a short wrapper, with the same name, say:
```c++
void TestDeserializerFromJson(string const & jsonBuffer, string const & name, vector<Obj> const & expected)
{
  return TestDeserializerFromJson(jsonBuffer, OsmIdToFeatureIdsMap{}, name, expected);
}
```
and get rid of these modifications of the old code?

Reply:
Done